<launch>
	<node pkg="toposens_pointcloud" type="toposens_pointcloud_node" name="ts_pointcloud_node" output="screen">
		<param name="octree_size" value="20.0" />
		<param name="octree_divisions" value="12.0" />
		<param name="fixed_frame" value="odom" />
	</node>

	<node pkg="rviz" type="rviz" name="grid_visualization" output="screen" args="-d $(find toposens_pointcloud)/rviz/robot_markers.rviz" />
</launch>
